modules/rosa-creating-node-tuning.adoc

```diff
@@ -1,6 +1,6 @@
 // Module included in the following assemblies:
 //
-// * rosa_hcp/rosa-tuning-config.adoc
+// * nodes/nodes/nodes-node-tuning-operator.adoc
```

Comment:
This module still exists in the following assemblies:

1. `nodes/nodes/nodes-node-tuning-operator.adoc`
2. `nodes/nodes/rosa-tuning-config.adoc`